git-extra/git-sdk.sh

```diff
@@ -45,6 +45,15 @@ sdk () {
 		EOF
 		;;
 	welcome)
+		test -z "$MSYS_NO_PATHCONV" || {
+			cat >&2 <<-EOF
+
+			WARNING: You have the MSYS_NO_PATHCONV env var defined
+			Consult documentation regarding potential consequences and advices
+
+			EOF
+		}
```

Comment:
Could I ask you to lose the superfluous curly brackets and indentation? Also, since this hints at some documentation, we should probably find a URL to link to.